docker/solr/Dockerfile

```diff
@@ -0,0 +1,82 @@
+
+FROM openjdk:11-jre
```

Comment:
@dpfitzsi any specific reasons to add the Dockefile?
Since we use an image, I don't think we need to re-declare the dockerfile.

Reply:
Discussed during stand up with @dpfitzsi, and the Dockerfile is not needed. He will provide an update.

Reply:
@mtem27 if Dan does not provide an update, please go ahead and remove the file tomorrow. Thanks!